src/runtime-rs/crates/hypervisor/src/dragonball/mod.rs

```diff
@@ -127,4 +128,8 @@ impl Hypervisor for Dragonball {
         let inner = self.inner.read().await;
         inner.get_jailer_root().await
     }
+
+    async fn save(&self) -> Option<HypervisorState> {
+        todo!()
```

Comment:
This will panic so if this is still to be implemented why not add a helpful message as a param?

```suggestion
        todo!("See https://github.com/kata-containers/kata-containers/issues/XXX")
```

Reply:
fixed